src/test/testdata/expected/lsl/shiftSquareBrackets.lsl

```diff
@@ -0,0 +1,24 @@
+libsl "1.0.0";
+library simple;
+types {
+    int(int32);
+}
+automaton a : int {
+    state S1;
+    state S2;
+    state S3;
+    shift S1 -> S2(foo);
+    shift S1 -> S2(fix, foo);
+    shift S1 -> S2(foo);
+    shift S1 -> S2(foo);
+    shift S1 -> S3(sum(int, int), min(int));
```

Comment:
It looks like you didn't update `dumpToString()` function to drop out old syntax

Reply:
Updated the function, but there is a question. Should we check  for trailing comma in functions list (print it out too) and absence of square brackets (if there is only one function)? 
Right now `dumpToString()` always prints square brackets and doesn't check for trailing comma.

Reply:
This always depends on you. I think the better way is to absence square brackets and to skip trailing comma